plugin/src/main/java/org/screamingsandals/bedwars/config/Configurator.java

```diff
@@ -641,6 +641,7 @@ public void createFiles() {
         checkOrSetConfig(modify, "party.enabled", false);
         checkOrSetConfig(modify, "party.autojoin-members", false);
         checkOrSetConfig(modify, "party.notify-when-warped", true);
+        checkOrSetConfig(modify, "hide-lobby-after-game-start", false);
```

Comment:
0.3.0 uses `invisible-lobby-on-game-start` instead of `hide-lobby-after-game-start`. I'd use the 0.3.0 naming, or at least have the same naming in both versions